js_modules/dagster-ui/packages/eslint-config/.eslintrc.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
   extends: './index.js',
+  overrides: {
+    rules: {
+      '@typescript-eslint/no-var-requires': 'off',
+    },
+  },
```

Comment:
Is this just the eslintrc for locally linting this package?

Reply:
yep. I should just disable typescript completely in this package since it's JS only but wanted to remove this rule specifically so I don't need to keep disabling the rule that complains about me using `require` instead of `import`